0008-security-recommendations/README.md

```diff
@@ -0,0 +1,98 @@
+---
+name: KP Security Recommendations
+shortname: 8/Security
+status: raw
+editor: Alexey Shmalko <ashmalko@cybervisiontech.com>
+---
+## Introduction
+This document describes security recommendations for systems using 1/KP. This is only a recommendation and is not mandatory to implement.
+
+## Model
+We start with the generalization of the system to better understand the attack surface, threats, and possible protection techniques.
+
+The general system consists of a server, clients, and end devices. End devices are devices that produce data, and the client only serves as a proxy for them. End devices are represented as endpoints within 1/KP.
+
+So the connection scheme looks like this:
+```
+server -- clients -- end devices
```

Comment:
s/end devices/endpoints ?
- here and below, because "end devices" may not be devices at all. Moreover, "endpoint" is already defined in 1/KP, so it's fine to use.